domains/client/domain-operator/src/domain_bundle_proposer.rs

```diff
@@ -29,7 +29,7 @@ const MAX_SKIPPED_TRANSACTIONS: usize = 8;
 const BUNDLE_UTILIZATION_THRESHOLD: Percent = Percent::from_percent(95);
 
 // `PreviousBundledTx` used to keep track of tx that have included in previous bundle and avoid
-// to re-include the these tx in the following bundle to reduce deplicated tx.
+// to re-include the these tx in the following bundle to reduce deprecated tx.
```

Comment:
`to re-including these transactions in the next bundle if the consensus hash did not change.`

Reply:
done